hydromt/data_adapter/geodataframe.py

```diff
@@ -33,7 +33,7 @@
 class GeoDataFrameAdapter(DataAdapterBase):
     """The GeoDataFrameAdapter performs transformations on GeoDataFrames."""
 
-    source: GeoDataFrameDataSource
+    source: GeoDataSource
     used: bool = False
```

Comment:
These are all class variables and should be object variables. They should be used to `def __init__()`